src/Updater.php

```diff
@@ -348,13 +355,21 @@ protected function downloadPhar()
             );
         }
 
-        if ($this->getStrategy() instanceof ShaStrategy) {
-            $tmpVersion = sha1_file($this->getTempPharFile());
+        if ($this->getStrategy() instanceof ShaStrategy
+        || $this->getStrategy() instanceof Sha256Strategy) {
```

Comment:
do you mind adding an indent in those case? I find hard to read otherwise (might just be me though)

```php
if ($this->getStrategy() instanceof ShaStrategy
    || $this->getStrategy() instanceof Sha256Strategy
) {
```